cmd/oci-create-runtime-bundle/main.go

```diff
@@ -114,6 +118,18 @@ func (v *bundleCmd) Run(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
+	//find string complying with regular expression `^[a-zA-Z0-9\-\.\_]+?$`, just for test
+	stringRefFind := refRegexp.FindString(v.ref)
+	v.stderr.Printf("ref name is %s", stringRefFind)
+
+	//check whether string ref match regular expression or not
+	boolRefMatch := refRegexp.MatchString(v.ref)
+	v.stderr.Printf("matching result is %v", boolRefMatch)
+	if boolRefMatch != true {
+		v.stderr.Printf("ref name (%s) is invalid", v.ref)
+		os.Exit(1)
+	}
```

Comment:
This probably belongs in image-layout validation like `image/image.go`'s `validate`.